life_expectancy/cleaning.py

```diff
@@ -11,9 +12,12 @@
 SAVE_FILE_NAME = "pt_life_expectancy.csv"
 
 
-def clean_data(region_name: str):  # pylint: disable=C0116
-
+def load_data() -> pd.DataFrame:  # pylint: disable=C0116
```

Comment:
You don't need to disable that pylint error. Adding docstrings to functions is an excellent practice.